spec/support/button_as_link_helpers.rb

```diff
@@ -0,0 +1,23 @@
+module ButtonAsLinkHelpers
+  RSpec::Matchers.define :have_button_as_link do |text, attrs = {}|
```

Comment:
I would expect to find a custom matcher in the RSpec config I think. i.e. in the spec_helper file. But that's just my opinion

Reply:
I think it makes more sense in this case to have it as a separate file (as with the other one), but open to a vote?

Reply:
Done